node.gyp

```diff
@@ -344,11 +344,8 @@
             'src/inspector_socket_server.h',
           ],
           'dependencies': [
-            'deps/v8_inspector/src/inspector/inspector.gyp:standalone_inspector',
-            'v8_inspector_compress_protocol_json#host',
           ],
```

Comment:
@bnoordhuis 

> Does it perhaps work by accident because the cctest target also has a dependency on it?

I think so. I suppose we should add it as a dependency to the `node` target too?
Can you explain what makes `cctest` to be always built? Maybe that should only happen when `make cctest` is actually executed?